devices/forms.py

```diff
@@ -237,7 +237,7 @@ class DeviceGroupFilterForm(FilterForm):
 
 class DeviceForm(forms.ModelForm):
     error_css_class = 'has-error'
-
+    uses = forms.MultipleChoiceField(choices = Device.objects.none(), required = False)
```

Comment:
Unfortunately, I have a headache with this approach.

In production we have > 4.5k devices with inconsistent names and a multiple choice drop down is pretty much unusable without a proper search. I think we need to reference the associated device by ID here.

Reply:
Do you mean the inventory number or the serial number? Or the internal id?

Reply:
Anything that works, really. The device ID is just the first thing that comes to mind.